src/ext-test/java/org/opentripplanner/ext/siri/SiriTimetableSnapshotSourceTest.java

```diff
@@ -40,10 +41,15 @@
 
 class SiriTimetableSnapshotSourceTest {
 
+  private RealtimeTestEnvironment env;
+
+  @BeforeEach
+  void setUp() {
+    env = new RealtimeTestEnvironment();
+  }
```

Comment:
This is more a matter of style so I will not insist, but I actually preferred @habrahamsson-skanetrafiken's way of instantiating. He can have the final say on it.

Reply:
The thing I like about having it in each test is that each test is self-contained and it makes it very easy to parametrize the environment, like if you want to setup some test with a different timezone for example.